packages/job-worker/src/playout/abPlayback/index.ts

```diff
@@ -6,35 +6,45 @@ import { WrappedShowStyleBlueprint } from '../../blueprints/cache'
 import { ReadonlyDeep } from 'type-fest'
 import { JobContext, ProcessedShowStyleCompound } from '../../jobs'
 import { getCurrentTime } from '../../lib'
-import { resolveAbAssignmentsFromRequests } from './abPlaybackResolver'
+import { resolveAbAssignmentsFromRequests, SessionRequest } from './abPlaybackResolver'
 import { calculateSessionTimeRanges } from './abPlaybackSessions'
 import { applyAbPlayerObjectAssignments } from './applyAssignments'
 import { AbSessionHelper } from './abSessionHelper'
 import { ShowStyleContext } from '../../blueprints/context'
 import { logger } from '../../logging'
 import { ABPlayerDefinition } from '@sofie-automation/blueprints-integration'
+import { PlayoutModel } from '../model/PlayoutModel'
+import { applyAndValidateOverrides } from '@sofie-automation/corelib/dist/settings/objectWithOverrides'
+import { StudioRouteSet } from '@sofie-automation/shared-lib/dist/core/model/StudioRouteSet'
+
+interface MembersOfRouteSets {
+	poolName: string
+	playerId: string | number
+	disabled: boolean
+}
 
 /**
  * Resolve and apply AB-playback for the given timeline
  * @param context Context of the job
  * @param abSessionHelper Helper for generation sessionId
  * @param blueprint Blueprint of the currently playing ShowStyle
  * @param showStyle The currently playing ShowStyle
- * @param playlist The currently playing Playlist
+ * @param playoutModel The current playout model
  * @param resolvedPieces All the PieceInstances on the timeline, resolved to have 'accurate' playback timings
  * @param timelineObjects The current timeline
  * @returns New AB assignments to be persisted on the playlist for the next call
  */
-export function applyAbPlaybackForTimeline(
+export async function applyAbPlaybackForTimeline(
```

Comment:
It doesn't look like the method signature changes are needed. (either playlist->playoutModel or making async).
would be good to undo those

Reply:
Has been undone, and as you say, it's not needed, with the current implementation.